app/util/concurrent/BackgroundProcessor.scala

```diff
@@ -36,8 +38,8 @@ object BackgroundProcessor {
 
   type SendType[T] = Tuple2[Option[Throwable], Option[T]]
   def send[PROC_RES,RESPONSE](cmd: BackgroundProcess[PROC_RES])(result: SendType[PROC_RES] => RESPONSE)(implicit mf: Manifest[PROC_RES]) = {
-    ask(ref, cmd)(cmd.timeout).mapTo[PROC_RES].asPromise.extend1 {
-      case Redeemed(v) => result(Tuple2(None, Some(v)))
+    ask(ref, cmd)(cmd.timeout).mapTo[PROC_RES].map {
```

Comment:
No, I can run an asset through intake without IPMI working on master, just not this branch.